src/genie/libs/parser/iosxe/show_lldp.py

```diff
@@ -698,7 +698,9 @@ def cli(self, output=None):
         # router               Gi1/0/52       117        R               Gi0/0/0
         # 10.10.191.107       Gi1/0/14       155        B,T             7038.eeff.572d
         # d89e.f3ff.58fe      Gi1/0/33       3070                       d89e.f3ff.58fe
-        p2 = re.compile(r'(?P<device_id>\S+)\s+(?P<interfaces>\S+)'
+        # Polycom Trio Visual+Gi2/0/28       120        T               6416.7f1e.ff12
+
+        p2 = re.compile(r'(?P<device_id>.{20})(?P<interfaces>\S+)'
```

Comment:
Is there any particular reason for choosing 20 or is it arbitrary? Could it possibly cause issues with other input?

Reply:
it is tabular data and the field width is 20. A tabular parser might have been better but since I was modifying existing regex this seemed the best choice to identify the device id field.